### PR TITLE
Fixes #85 - Added integration test for deprecation headers.

### LINKED ISSUE #85: Add integration test for deprecation headers
Refs #82 which adds contextual test server configuration, so we now should add integration tests for deprecation headers.

### PR DESCRIPTION
**Pull request by n1k0**
r=? @Natim

### CODE REVIEW

**Comment by n1k0** [comment]
Hmm I should probably add a test for hard EOL. Hold on.

File: test/integration_test.js [hunk 1]
```diff
@@ -320,4 +325,27 @@ describe("Integration tests", () => {
...
+        CLIQUET_EOS_URL: "http://www.perdu.com",
+        CLIQUET_EOS_MESSAGE: "Boom",
+      });
+    });
+
+    after(() => {
+      return stopServer();
+    });
+
+    it("should reject sync when the server sends a Backoff header", () => {
```
Comment (Natim) [inline test/integration_test.js]: I think this test is not about Backoff header

**Comment by n1k0** [comment]
Now with Hard EOL test added. r=? @Natim

**Comment by Natim** [comment]
r+

**Pull request merged**